packages/eslint-plugin-fiori-tools/src/rules/sap-browser-api-warning.ts

```diff
@@ -0,0 +1,396 @@
+        }
+        /**
+         *
+         * @param node
+         */
+        function isMember(node: any) {
+            return isType(node, MEMBER);
+        }
+        /**
+         *
+         * @param node
+         */
+        function isCall(node: any) {
+            return isType(node, CALL);
+        }
+        /**
+         *
+         * @param node
+         */
+        function isCondition(node: any) {
+            return isType(node, IF_CONDITION) || isType(node, CONDITION_EXP);
+        }
+        /**
+         *
+         * @param node
+         */
+        function isUnary(node: any) {
+            return isType(node, UNARY);
+        }
+        /**
+         *
+         * @param node
+         */
+        function isLiteral(node: any) {
+            return isType(node, LITERAL);
+        }
+
+        /**
+         *
+         * @param a
+         * @param obj
+         */
+        function contains(a, obj) {
+            for (let i = 0; i < a.length; i++) {
+                if (obj === a[i]) {
+                    return true;
+                }
+            }
+            return false;
+        }
+
+        /**
+         *
+         * @param methodName
+         */
+        function isDomAccess(methodName) {
+            return contains(FORBIDDEN_DOM_ACCESS, methodName);
+        }
+
+        /**
+         *
+         * @param methodName
+         */
+        function isWindowUsage(methodName) {
+            return contains(FORBIDDEN_WINDOW_USAGES, methodName);
+        }
+
+        /**
+         *
+         * @param node
+         */
+        function isWindow(node: any): boolean {
+            return isIdentifier(node) && 'name' in node && node.name === 'window';
+        }
+
+        /**
+         *
+         * @param node
+         * @param justHistory
+         */
+        function isHistory(node: any, justHistory: any): boolean {
+            if (isIdentifier(node) && 'name' in node) {
+                return node.name === 'history' || (!justHistory && contains(FORBIDDEN_HISTORY_OBJECT, node.name));
+            } else if (isMember(node)) {
+                return isWindow(node.object) && isIdentifier(node.property) && isHistory(node.property, true);
+            }
+            return false;
+        }
+
+        /**
+         *
+         * @param node
+         */
+        function getRightestMethodName(node: any) {
+            if (isMember(node.callee)) {
+                return node.callee.property.name;
+            } else {
+                return node.callee.name;
+            }
+        }
+
+        /**
+         *
+         * @param node
+         */
+        function processVariableDeclarator(node: any) {
+            if (node.init) {
+                if (isMember(node.init)) {
+                    let firstElement = node.init.object.name;
+                    const secondElement = node.init.property.name;
+
+                    if (firstElement + '.' + secondElement === 'window.document') {
```

Comment:
General remark for string concatenation

```suggestion
                    if (`${firstElement}.${secondElement}` === 'window.document') {
```